src/main/java/com/perficient/movies/model/entities/Artist.java

```diff
@@ -9,7 +9,6 @@
 @Entity
 @Inheritance(strategy = InheritanceType.JOINED)
```

Comment:
Why do you use inherence Joined? What is the reason?

Reply:
Because it builds a database model with separate tables but, connected by its ids. 
My first experiment was with _@MappedSuperclass_ , the result was two tables with all the attributes, however, my objective was 4 tables. That's why I looked another way.

![image](https://user-images.githubusercontent.com/42673790/154580269-dfec0d2a-7e12-487f-aff4-0cec7609d22e.png)